README.md

```diff
@@ -80,25 +80,98 @@ Syntax highlighting is done using semantic tokens. This is mainly to avoid regex
+-   A Lisp tree view is added that shows the REPL history, threads, packages, and defined ASDF systems.
 -   History items can be re-run by using the up/down arrow keys in the REPL console, using the re-run action in the history tree view, or using "REPL History" from the command palette.
 
-## Inspector
+### Lisp Tree View
 
-An inspector can be opened by evaluating an expression in the inspector view or by clicking `Inspect` at the bottom of the hover text for a symbol.
+The Lisp Tree View is invoked by clicking on the following icon in the Activty Bar on the left of the VSCode workbench:
 
-There is also an inspector for macros, using the Inspect Macro command. An inspector opens that shows one level of expansion for the macro at the current cursor position. It has a button to refresh the inspector or increment the level of expansion by one. When an expression is sent for evaluation, such as redefining the macro, the expansion is refreshed back to one level.
+<img src="images/lisp-icon-sm.png" alt="alt text"/>
 
-### Eval Text
+There are a number of panels in the Lisp Tree View to display and
+manipulate different aspects of your current Lisp REPL.
 
-At the bottom of each inspector view is a text field that can be used to evaluate expressions.
+#### REPL History
+
+The **REPL HISTORY** view displays all of the Lisp forms executed in the provided REPL.
+The remove icon at the right of the title bar for this view clears the history.
+
+For each form in the list there are actions.
+Clicking on a form expands to show the form's containing package.
+To the right of the form are three icons:
+* The pencil icon puts the form into the entry line below the REPL
+  and leaves it there to be edited and executed.
+* The circle arrow icon sends the form to the REPL to be executed again.
+* The minus icon removes the item from the history.
+
+#### Inspector
+
+The **INSPECTOR** view can generate an inspect panel for a symbol.
+Inspection is done in the context of a specific package.
+The line below the title bar displays the current package,
+clicking on that line brings up a menu of packages from which to choose.
+
+Once the package is chosen, the next line is user entry box
+for the name of the symbol to be inspected.
+Once a symbol is entered an inspect panel is generated on the right side of the workbench.
+By default unquoted symbols are variables in the chosen package.
+To inspect a function it is necessary to quote it by prepending the name with `'`.
+
+#### Packages
+
+The **PACKAGES** view shows all of the packages in the REPL.
+The circle arrow icon at the right of the title bar for this view
+updates the package list to be current with the REPL,
+this view is not automatically updated when packages are added.
+
+For each package in the list there are actions.
+Clicking on the minus icon at the right removes the package.
+Clicking on the package itself expands to a list of symbols in that package.
```

Comment:
Since the Packages view drills down into package symbols, have you considered adding an inspect button to each symbol item? That might be easier than the way the Inspector view works.

Reply:
Yeah, I've had some grandiose ideas for the tree view. :-) The main idea is that it should be an active view into the running image. Having an inspector button would be good. I've also thought of having navigation like the file tree. You click on a symbol and it jumps to where it's defined. Stuff like that.

The current inspector thing is so you can enter any lisp expression and it'll open an inspector with the result. There's an inspect link at the bottom of the hover for symbols, but I'm not sure anybody ever sees it.